### PR TITLE
finished project - without comment changes

### PR DESCRIPTION
**Pull request by Aqpogthy**

### CODE REVIEW

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                {                                 
+                    break;
+                }
+                Console.WriteLine("Please enter 1 or 2");
+            }
+            
+            
+            List<string> text = new List<string>();
+            if (option == "1") //See if the user chose to enter the sentances manually
+            {
```
Comment (Yumeyums) [inline CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs:36]: Sentence mispelled here, consistent throughout so it doesn't affect the code running though.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+            //Create an 'Analyse' object
+            Analyse AnalyseText = new Analyse();
+            //Pass the text input to the 'analyseText' method and receive a list of integers back
+            parameters = AnalyseText.analyseText(text);
+
+            //Report the results of the analysis
+            Report SentanceReport = new Report();
+            SentanceReport.OutputAnalsis(parameters, AnalyseText.GetFrequency());
```
Comment (Yumeyums) [inline CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs]: Analysis misspelt here as well, could affect future outputs, may need to correct it.

**Review by Yumeyums — COMMENTED** [review]
There were a bit of misspelled words in the code that may need fixing, overall good work, the code comments provide adequate explanations of what is going on, for the vowels and consonants however, to make them shorter, I recommend using an array to store the vowel characters in order to iterate through them better, rather than using an or across all of them in one if statement.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Input.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        string text = "nothing";
+        
+        //Method: manualTextInput
+        //Arguments: none
+        //Returns: string
+        //Gets text input from the keyboard
+        public string manualTextInput()
+        {
+            Console.WriteLine("Please enter your sentance to be analysed. You should input one line at a time. (End your last sentance with *)");
+            string text = Console.ReadLine();
```
Comment (Beniial) [inline CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Input.cs:21]: The message states that you should input sentences one line at a time, but only reads the console once, making this redundant.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Report.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        public void OutputAnalsis(List<int> parameters, Dictionary<char,int> Frequency)
+        {
+            Console.WriteLine("Number of sentences entered = " + parameters[0]);
+            Console.WriteLine("Number of vowels = " + parameters[1]);
+            Console.WriteLine("Number of consonants = " + parameters[2]);
+            Console.WriteLine("Number of upper case letters = " + parameters[3]);
+            Console.WriteLine("Number of lower case letters = " + parameters[4]);
+            foreach (var letterfrequence in Frequency) 
+            {
+                Console.WriteLine(letterfrequence.Key + ":" + letterfrequence.Value);
```
Comment (Beniial) [inline CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Report.cs:24]: Context is not given as to what these values mean when printed, unlike the rest that clearly state "Number of vowels", "Number of sentences", etc.

**Review by Beniial — COMMENTED** [review]
The code is mostly well documented, but certain sections and variables don't properly explain their purpose. Error handling appears in place, but may not be able to handle edge cases, such as ellipsis or interrobangs, which would be marked as multiple sentences. There are also a number of typos and aesthetic mistakes, however these do not ultimately hurt the performance of the program.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Input.cs (hunk 3 above)
Comment (Aqpogthy) [reply CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Input.cs:21]: This function is repeatedly passed through in the Main, unitl the last character in the string is a '*'.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs (hunk 1 above)
Comment (Aqpogthy) [reply CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs:36]: All sentence misspells have been replaced.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs (hunk 2 above)
Comment (Aqpogthy) [reply CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs]: All analysis misspells have been replaced.

File: CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Report.cs (hunk 4 above)
Comment (Aqpogthy) [reply CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Report.cs:24]: I have added this to my code. Thank you for pointing this out to me.

**Comment by Aqpogthy** [comment]
> The code is mostly well documented, but certain sections and variables don't properly explain their purpose. Error handling appears in place, but may not be able to handle edge cases, such as ellipsis or interrobangs, which would be marked as multiple sentences. There are also a number of typos and aesthetic mistakes, however these do not ultimately hurt the performance of the program.

In my Analyse Class I checked to see if a sentance happened to be 1 character long or shorter. This prevents problems from ellipses or interrobangs. I have added a comment stating the purpose of the if statement.

**Comment by Aqpogthy** [comment]
> There were a bit of misspelled words in the code that may need fixing, overall good work, the code comments provide adequate explanations of what is going on, for the vowels and consonants however, to make them shorter, I recommend using an array to store the vowel characters in order to iterate through them better, rather than using an or across all of them in one if statement.

Thank you for your review. I have altered how I check if a letter is a vowel.

**Pull request merged**